tracer/src/Datadog.Trace/Tracer.cs

```diff
@@ -145,6 +145,11 @@ public static Tracer Instance
                       " instance in code.")]
             set
             {
+                if (value is null)
+                {
+                    ThrowHelper.ThrowArgumentNullException("The tracer instance shouldn't be set to null as this would cause issues to the automatic instrumentation.");
```

Comment:
```suggestion
                    ThrowHelper.ThrowArgumentNullException("The tracer instance shouldn't be set to null as this will cause issues with automatic instrumentation.");
```

Reply:
Don't let a French guys fix your messages, because later someone else is going to fix it :) (or at least don't let me do it, let's not put all French people in the same bag :p)